pkg/quarterdeck/permissions/permissions.go

```diff
@@ -0,0 +1,34 @@
+package permissions
+
+// These permissions are used to authorize user requests and should match the defined
+// permissions in the quarterdeck database.
+const (
+	// Organizations management
+	CreateOrganizations = "organizations:create"
```

Comment:
I don't think we need `organizations:create` do we? Anyone can create an organization even without be authenticated (e.g. when you first register).